### ISSUE TITLE
Skunkworks Project Suggestion: 

### ISSUE THREAD

**Issue by Samoris** [scroll-tech/contribute-to-scroll#272]
### Name of the project

Hidden fees on scroll

### Project abstract

We want the team to be transparent with the users on fees charged, we get charged way more than what we approve of on fees while signing a transaction 

### Specification

Please look into this, it would be better if it was visible to us, and we know this is exactly what we will pay as fees, and we sign it if we are comfortable with it

**Comment by Iamprime0** [comment]
Very annoying. Remembered when I couldn't mint the scroll NFT because of this same issue. It should be visible or stated from onset, that we're going to be paying extra fees, instead of hiding it and making a nuisance of our hard earned money. The team should work on that please.

**Comment by chuksl** [comment]
this sucks for real,i hope the team works on it soon

**Comment by Adelekejr** [comment]
Finally someone is asking the right question. I mean there should be an explanation why it's that much. You can't make do with little unless you get more token. We're not all the same financially.
I hope they respond

**Comment by rickdeegithub** [comment]
Initially, I suspected it was my gas settings, but then I realized others were experiencing the same issue. Despite the gas fee showing as $0.2 - $0.3, I ended up being charged over $2 for each transactions, even when the gas price was low. It's frustrating that they're not transparent about their charges—I can't stay on top of this.

**Comment by Tophhhh** [comment]
this sucks for real,i hope the team works on it soon
Fr

**Comment by Icekreem01** [comment]
It’s a great project going on here

**Comment by Ollypumpay01** [comment]
Yeah, I mean this is web3 and transparency is key. The amount we’re paying should be defined before we execute transactions.

**Comment by muideenad24** [comment]
that is really bad fr
